static/app/types/jsSelfProfiling.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for https://wicg.github.io/js-self-profiling/
+namespace JSSelfProfiling {
```

Comment:
We actually haven't usually used namespaces pretty much at all. But I wonder if we should start?

Good topic for the TSC

Reply:
@evanpurkhiser I think here it makes sense, as the terminology is often times very similar, it just varies across formats and this makes it easier to know where it belongs :)